client/packages/invoices/src/InboundShipment/DetailView/SidePanel/PricingSection.tsx

```diff
@@ -98,7 +98,7 @@ export const PricingSectionComponent = () => {
           <PanelLabel>{`${t('heading.tax')} ${Formatter.tax(tax)}`}</PanelLabel>
           <PanelField>
             <TaxEdit
-              disabled={false}
+              disabled={!stockLines?.length}
```

Comment:
I think that if the shipment is verified, we shouldn't be able to edit service changes or tax details. This is how mSupply works and I'm taking the logic partly from there 🤷

Reply:
Agreed. Fixed by: [8f50ebd](https://github.com/openmsupply/open-msupply/pull/1311/commits/8f50ebdb752388e9f457833fecadca2c22673d37)